src/components/Home.jsx

```diff
@@ -0,0 +1,37 @@
+import { useSelector } from 'react-redux';
+import RecipeCard from './RecipeCard';
+import recipeLogo from '../assets/img/recipe.svg';
+
+const Home = () => {
+  const { recipes, isLoading } = useSelector((store) => store.recipes);
+  const { search } = useSelector((store) => store.navbar);
+
+  if (isLoading) {
+    return (
+      <div className="loading">
+        <p>Loading recipes...</p>
+      </div>
+    );
+  }
+
+  return (
+    <div className="home">
+      <div className="headline">
+        <img className="logo" src={recipeLogo} alt="logo" />
+        <p className="headline-text">Here is a list of the 20 best recipes for you to enjoy</p>
+      </div>
+      <div className="divider">Number of ingredients required</div>
+      {recipes.filter((recipe) => (search.toLowerCase() === '' ? recipe : recipe.label.toLowerCase().includes(search))).map((recipe) => (
```

Comment:
As requested by the previous reviewer, kindly make sure that there's a functionality that filters categories on the homepage.

- The could be a search bar that allows users to search for a specific recipe on the homepage or a dropdown that allows users to filter the recipes by groups (i.e breakfast, smoothies, desserts).

You can put the search bar here:

![image](https://user-images.githubusercontent.com/38088818/233454331-96af0a3c-c5ad-4e15-a025-22e7ef240ffc.png)

Reply:
Sorry bro, but I'm going to have to give you a bad review, the search bar is implemented. Had you looked at the components you'd have seen that I did add the search feature on the navbar component.

Reply:
search filter added good job  :+1: 
![Screenshot from 2023-04-20 20-59-44](https://user-images.githubusercontent.com/66097371/233475134-4b366044-bf51-441a-a000-b7845127378d.png)